elastic-search-support/src/main/resources/application.yml

```diff
@@ -51,6 +51,36 @@ elasticsearch:
                             }
                         }
                     }'
+    ccdDynamicRadioList: '{
+                            "properties": {
+                                "value": {
+                                    "properties": {
+                                        "code": {
+                                          "type": "text" , "fields": { "keyword": { "type": "keyword", "ignore_above": 256}}
+                                        },
+                                        "label": {
+                                          "type": "text" , "fields": { "keyword": { "type": "keyword", "ignore_above": 256}}
+                                        }
+                                    }
+                                },
+                                    "list_items": {
+                                      "type": "object",
+                                      "enabled": false
+                                }
+                            }
+                        }'
+    ccdDynamicMultiSelectList: '{
+                            "properties": {
+                                "value": {
+                                    "type": "object",
+                                    "enabled": false
+                                },
```

Comment:
this seems wrong. With this config we won't be able to search cases that have a certain value for a dynamic multiselect field. 
To understand mapping rules for arrays, see: https://www.elastic.co/guide/en/elasticsearch/reference/current/array.html

I think we could just do this:
```
DynamicRadioList: ${elasticsearch.elasticMappings.ccdDynamicList}
DynamicMultiSelectList:  ${elasticsearch.elasticMappings.ccdDynamicList}

and remove  ccdDynamicRadioList and ccdDynamicMultiSelectList since those will always be identical to ccdDynamicList